src/components/GroceryItem/GroceryItem.js

```diff
@@ -47,16 +48,49 @@ const GroceryItem = ({ item }) => {
       });
   };
 
-  const resetDate = () => {
+  // Should we delete the option to uncheck? Otherwise, we can't really reset the previousPurchaseDate in Firestore.
+  const handleUncheckBox = () => {
     setChecked(false);
   };
 
+  const getAriaLabel = () => {
+    if (
+      currentDate - item.purchaseDate >=
+        2 * (item.nextPurchaseDate - item.purchaseDate) ||
+      item.numberOfPurchases === 0
+    ) {
+      return 'Inactive';
```

Comment:
What do you think about changing this text for something more descriptive? Probably that users would appreciate having more context on what "inactive" means here, but I'd like to hear your thoughts on this.